Python-Function_POW.md

```diff
@@ -0,0 +1,21 @@
+# Python 'pow(x,y)'
+
+pow(x, y[, z]) is a built in function in Python 3 to calculate x to the power y and if z is present , returns x to the power y modulo [https://processing.org/reference/modulo.html] z
+
+#Argument 
+The arguments must have numeric types.
+It takes two argument x and y as well as three x,y and z.
```

Comment:
~~It takes two argument x and y as well as three x,y and z.~~ This function takes two arguments, `x` and `y`, as well as three, `x`,`y` and `z`.